include/alpaka/test/KernelExecutionFixture.hpp

```diff
@@ -25,6 +25,13 @@ namespace alpaka::test
     template<typename TAcc>
     class KernelExecutionFixture
     {
+#if defined(BOOST_COMP_GNUC) && BOOST_COMP_GNUC >= BOOST_VERSION_NUMBER(11, 0, 0)                                     \
+    && BOOST_COMP_GNUC < BOOST_VERSION_NUMBER(12, 0, 0)
+// g++-11 (wrongly) believes that m_platformHost is used in an uninitialized state.
+#    pragma GCC diagnostic push
+#    pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#endif
```

Comment:
That's a shame! And the braces in `PlatformCpu m_platformHost{};` really don't solve it?

Reply:
Nope :-(